src/main/java/com/couchbase/lite/router/Router.java

```diff
@@ -375,6 +375,14 @@ private static String getAccept(URLConnection connection) {
         return accept;
     }
 
+    private static String getIfMatch(URLConnection connection) {
+        String ifMatch = connection.getRequestProperty("If-Match");
+        if (ifMatch == null)
+            // From Android: http://developer.android.com/reference/java/net/URLConnection.html
+            ifMatch = connection.getRequestProperty("if-match");
+        return ifMatch;
+    }
+
```

Comment:
I have noticed that you started to wrap connection.getRequestProperty() call with normal case and lowercase property name. I have seen a lot of connection.getRequestProperty() in Router.java. Do all connection.getRequestProperty() calls need a wrapper like this? If YES, it is better to have one method to do this instead of one per property.